Makefile

```diff
@@ -0,0 +1,11 @@
+OUT_FILE := ./bin/e2e_appstudio
+DOCKER_IMAGE_NAME :=quay.io/flacatus/e2e:next
```

Comment:
Pls change this to `quay.io/redhat-appstudio/e2e`
I can work on getting access there and providing some basic github actions for this repo (linter, build&push to quay, ...)